js/index.js

```diff
@@ -38,5 +43,83 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+let logo = document.getElementById('logo-img');
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+
+
+let navLinks = document.querySelectorAll('nav a');
+navLinks[0].textContent = siteContent['nav']['nav-item-1'];
+navLinks[1].textContent = siteContent['nav']['nav-item-2'];
+navLinks[2].textContent = siteContent['nav']['nav-item-3'];
+navLinks[3].textContent = siteContent['nav']['nav-item-4'];
+navLinks[4].textContent = siteContent['nav']['nav-item-5'];
+navLinks[5].textContent = siteContent['nav']['nav-item-6'];
+
+
+let title = document.getElementsByClassName('cta-text')[0];
+//title.textContent = siteContent['cta']['h1'];
+
+title.getElementsByTagName('h1')[0].innerHTML = siteContent['cta']['h1'];
+title.getElementsByTagName('button')[0].innerHTML = siteContent['cta']['button'];
+
+
+let headImg = document.getElementById('cta-img');
+headImg.setAttribute('src', siteContent['cta']['img-src'])
+
+
+let textContent = document.querySelectorAll('.text-content');
+textContent[0].getElementsByTagName('h4')[0].innerHTML = siteContent['main-content']['features-h4'];
+textContent[0].getElementsByTagName('p')[0].innerHTML = siteContent['main-content']['features-content'];
+
+textContent[1].getElementsByTagName('h4')[0].innerHTML = siteContent['main-content']['about-h4'];
+textContent[1].getElementsByTagName('p')[0].innerHTML = siteContent['main-content']['about-content'];
+
+let midImg = document.getElementById('middle-img');
+midImg.setAttribute('src', siteContent['main-content']['middle-img-src'])
```

Comment:
you will notice that because the formatting is dependent on the pre made css and html, the JS is just inserting the object into the those locations. So Cascading placement is more semantic in the JS file.